src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -56,8 +56,9 @@ def generate(judge_model_name, branch, taxonomy_dir, output_dir):
                 print(f"failed to load {qna_file}. skipping...")
                 continue
             for ex in examples:
-                q, a = ex["question"], ex["answer"]
-                if q is None or a is None:
+                try:
+                    q, a = ex["question"], ex["answer"]
+                except KeyError:
```

Comment:
Before the change, `question` and `answer` were allowed / expected to be `Nones` (in which case we `continue`d.) Do we still have to do it?

Reply:
I see. So two different things. First to make sure the keys are present, and also that they are not empty.

Reply:
> Before the change, question and answer were allowed / expected to be Nones (in which case we continued.) Do we still have to do it?

> I see. So two different things. First to make sure the keys are present, and also that they are not empty.

You don't have to check for both conditions.  But it might be best to switch to .get instead of [] and only do the None check.

Reply:
Agreed. Was also just thinking that. Updating...